frontend/app/helpers/search_helper.rb

```diff
@@ -151,6 +162,49 @@ def add_browse_columns(model, enum_locales = {})
     end
   end
 
+  def ids_to_identifier(field)
+    id0 = JSONModel::HTTP.get_json(field)['id_0'] ? (JSONModel::HTTP.get_json(field)['id_0']).to_s : nil
+    id1 = JSONModel::HTTP.get_json(field)['id_1'] ? (JSONModel::HTTP.get_json(field)['id_1']).to_s : nil
+    id2 = JSONModel::HTTP.get_json(field)['id_2'] ? (JSONModel::HTTP.get_json(field)['id_2']).to_s : nil
+    id3 = JSONModel::HTTP.get_json(field)['id_3'] ? (JSONModel::HTTP.get_json(field)['id_3']).to_s : nil
+    [id0, id1, id2, id3].compact.join('.')
+  end
+
+  def get_ancestor_title(field)
+    if field.include? 'resources'
+      clean_mixed_content(JSONModel::HTTP.get_json(field)['title'])
+    elsif field.include? 'digital_objects'
```

Comment:
@lorawoodford Could this be combined with the if statement above like this: 
if field.include? 'resources' || field.include? 'digital_objects'
  clean_mixed_content(JSONModel::HTTP.get_json(field)['title'])
else
  clean_mixed_content(JSONModel::HTTP.get_json(field)['display_string'])
end